### PR TITLE
Flourishes

### PR DESCRIPTION
**Pull request by patrickbower**
Includes
- Flourish elements as groups
- Handlebars helper added to combine `presentation-svg` and `flourish` partials with `shapes.json`
- Front Matter controlled

closes #115

### CODE REVIEW

File: src/sass/_flourish-pullquotes.scss [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+.flourish-pullquote {
+  height: auto;
+  
+  &__text {
+    display: inline-block;
+    font-weight: 400;
+    font-style: italic;
+    font-size: 20px;
+  }
```
Comment (danbull) [inline src/sass/_flourish-pullquotes.scss]: I feel like we should stick with the gravity `blockquote` component or if it needs to be adjusted then that can be overridden here. Maybe one for @james-nash and @ThePeach to discuss.

File: templates/partials/pullquote.hbs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<div class="flourish-pullquote-wrap">
+  <div class="flourish flourish-pullquote">
+     {{#each shapes}}
+      <div class="flourish-pullquote__{{shape}} {{#if reveal}}js-reveal-element{{/if}}" {{#if exclude}}data-exclude="{{exclude}}"
+        {{/if}}>
+        {{ flourishShapes shape }}
+      </div>
+    {{/each}}
+    <span class="flourish-pullquote__text">
```
Comment (danbull) [inline templates/partials/pullquote.hbs]: Semantically this should be a `blockquote`?

Comment (danbull) [inline templates/partials/pullquote.hbs]: I'm really liking this much more now I've seen it on my laptop 👍I think it needs adjusting a bit on mobile though. This flourish seems too tall maybe? Could look better without the top padding too.

image

Also the footer flourish looks a bit too abstract and could look better with little or no padding underneath it.

image

File: gulp/metalsmith.js [hunk 3]
```diff
@@ -42,6 +55,10 @@ function metalsmith() {
     }
     return null;
   });
+  handlebars.registerHelper("flourishShapes", function(name, options) {
```
[the remark this answers is not in the document]
Comment (patrickbower) [reply gulp/metalsmith.js]: 👍

File: src/sass/_flourish-pullquotes.scss (hunk 1 above)
Comment (patrickbower) [reply src/sass/_flourish-pullquotes.scss]: Used `blockquote` element but that's all as flourish components are only on the website, not within Gravity.

File: templates/partials/pullquote.hbs (hunk 2 above)
Comment (patrickbower) [reply templates/partials/pullquote.hbs]: 👍

Comment (patrickbower) [reply templates/partials/pullquote.hbs]: Made these smaller and adjust the layout to reduce the weight on the page

**Comment by ThePeach** [comment]
as a side note, @patrickbower make sure you run this through different browsers so we don't have to come back to it after.

**Pull request merged**